src/Frontend/src/components/audit/FiltersPanel.vue

```diff
@@ -62,7 +60,7 @@ function findKeyByValue(searchValue: string) {
       <div class="filter-label"></div>
       <div class="filter-component text-search-container">
         <FilterInput v-model="messageFilterString" placeholder="Search messages..." aria-label="Search messages" />
-        <div class="note">Use <FAIcon :icon="faAsterisk" size="xs" /> as a wildcard. E.g.: <FAIcon :icon="faAsterisk" size="xs" />World! or Hello<FAIcon :icon="faAsterisk" size="xs" /> finds Hello World!</div>
+        <div class="note">Check the <a href="https://docs.particular.net/servicepulse/all-messages#filtering-options">documentation</a> to see the available operators and wildcards</div>
```

Comment:
I would simplify it to

```suggestion
        <div class="note">Check the <a href="https://docs.particular.net/servicepulse/all-messages#filtering-options">documentation</a> to see the available filter options</div>
```

or

```suggestion
        <div class="note">Check the <a href="https://docs.particular.net/servicepulse/all-messages#filtering-options">documentation</a> to see the available filtering options</div>
```